packages/cli/src/lib/container/docker_helper.ts

```diff
@@ -68,46 +69,120 @@ export const pullImage = function (resource: string) {
 
 export const getDockerfiles = function (rootdir: string, recursive: boolean) {
   const match = recursive ? './**/Dockerfile?(.)*' : 'Dockerfile*'
-  let dockerfiles: string[] = globSync(match, {
+  let dockerfiles = glob.sync(match, {
     cwd: rootdir,
     nodir: true,
   })
+
   if (recursive) {
-    dockerfiles = dockerfiles.filter((df: string) => df.match(/Dockerfile\.[\w]+$/))
+    dockerfiles = dockerfiles.filter(df => df.match(/Dockerfile\.[\w]+$/))
```

Comment:
Any reason we removed the typing here?

Reply:
It's automatically inferred as the `dockefiles` array is typed.